app/styles/_editor.less

```diff
@@ -50,7 +50,9 @@
   text-align: right;
 }
 
-#from-file .editor {
-  height: 350px;
+#from-file,
+#yaml-file {
```

Comment:
It looks like this rule with apply to Edit YAML action, not just Import YAML? Edit YAML currently grows with the page.